owncloudApp/src/main/java/com/owncloud/android/utils/FileStorageUtils.java

```diff
@@ -242,14 +242,12 @@ public static RemoteFile fillRemoteFile(OCFile ocFile) {
             val = -1;
         }
 
-        Collections.sort(files, new Comparator<OCFile>() {
-            public int compare(OCFile o1, OCFile o2) {
-                if (o1.getModificationTimestamp() == 0 || o2.getModificationTimestamp() == 0) {
-                    return 0;
-                } else {
-                    Long obj1 = o1.getModificationTimestamp();
-                    return val * obj1.compareTo(o2.getModificationTimestamp());
-                }
+        Collections.sort(files, (ocFile1, ocFile2) -> {
```

Comment:
You are not using `import java.util.Comparator;` anymore, are you?

Reply:
Removing it..